### PR TITLE
chore(frontend): Remove deprecated colors

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

A few colors were deprecated with the new mapping, so we clean up a bit, re-using the Design map where necessary.

### CODE REVIEW

File: src/frontend/src/lib/components/signer/SignerOrigin.svelte [hunk 1]
```diff
@@ -34,7 +34,7 @@
 {#if nonNullish(origin)}
 	<p class="mb-6 break-normal text-center">
 		{$i18n.signer.origin.text.request_from}
-		{#if nonNullish(host)}<span class="font-bold text-brandeis-blue"
+		{#if nonNullish(host)}<span class="font-bold text-foreground-brand-primary"
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/signer/SignerOrigin.svelte:37]: This information is particularly important. Can you provide a screenshot - i.e. have you double checked it is visually still predominent?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/signer/SignerOrigin.svelte:37]: This is the screenshot for that specific one

Screenshot 2024-11-01 at 08 32 58

Comment (peterpeterparker) [reply src/frontend/src/lib/components/signer/SignerOrigin.svelte:37]: Thx!

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**